### PR TITLE
terrascan: fix test

### PR DESCRIPTION
**Pull request by branchv**
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/HEAD/CONTRIBUTING.md)?
- [x] Have you ensured that your commits follow the [commit style guide](https://docs.brew.sh/Formula-Cookbook#commit)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Is your test running fine `brew test <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Does your build pass `brew audit --strict <formula>` (after doing `brew install --build-from-source <formula>`)? If this is a new formula, does it pass `brew audit --new <formula>`?

-----
Seen in #78666

### CODE REVIEW

File: Formula/terrascan.rb [hunk 1]
```diff
@@ -35,14 +35,14 @@ def install
...
-      \tPolicies Validated  :\t203
+      \tPolicies Validated  :\t\\d+
       \tViolated Policies   :\t0
       \tLow                 :\t0
       \tMedium              :\t0
       \tHigh                :\t0
     EOS
 
-    assert_match expected, shell_output("#{bin}/terrascan scan -f #{testpath}/ami.tf -t aws")
+    assert_match(/#{expected}/, shell_output("#{bin}/terrascan scan -f #{testpath}/ami.tf -t aws"))
```
Comment (nandahkrishna) [inline Formula/terrascan.rb:45]: Can we change this to use 2 `assert_match`es instead? One that matches the text for everything except "Policies Validated" and another that uses a regex for "Policies Validated".

Comment (nandahkrishna) [reply Formula/terrascan.rb:45]: Something like:
```ruby
expected = <<~EOS
  \tViolated Policies   :\t0
  \tLow                 :\t0
  \tMedium              :\t0
  \tHigh                :\t0
EOS

output = shell_output("#{bin}/terrascan scan -f #{testpath}/ami.tf -t aws")
assert_match expected, output
assert_match(/Policies Validated:\s+\d+/, output)
```

**Comment by nandahkrishna** [comment]
The audit failure seems to be because the homepage is using Cloudflare. We could consider using the GitHub repository as the homepage. (Was seen in #77325).

File: Formula/terrascan.rb (hunk 1 above)
Comment (branchvincent) [reply Formula/terrascan.rb:45]: certainly!

**Review by SMillerDev — APPROVED** [review]

**Pull request merged**